test/remote/gateways/remote_braintree_blue_test.rb

```diff
@@ -17,6 +17,14 @@ def setup
     }
   end
 
+  def test_supports_network_tokenization_brands
```

Comment:
Shouldn't we be testing the public interface instead? (`supports_network_tokenization?`)